USING.md

```diff
@@ -1,116 +1,216 @@
-strange error in a DNS configuration that is hard to duplicate - then this
-saved file can be sent to a developer that can take a closer look into the
-issue.
+The same test as above with the three options included:
 
-**Only run specific test**
+```sh
+docker run -t --rm zonemaster/cli zonemaster.net --no-ipv6 --show-testcase --locale=da
+```
+```sh
+zonemaster-cli zonemaster.net --no-ipv6 --show-testcase --locale=da_DK.UTF-8
+```
 
-If you only want to run a specific test rather than the whole suite of
-tests, you can do that as well. If you want to see if the name servers for
-a domain are in enough different ASNs, you can run this test case:
+The difference between running `zonemaster-cli` on Docker or local installation
+is the invocation string, `docker run -t --rm zonemaster/cli` vs.
+`zonemaster-cli`. To simplify this document, from now on the shorter
+`zonemaster-cli` will be used and for Docker the longer string will be assumed.
+To simplify repeated invocation on Docker an alias can be created for the shell.
 
-    $ zonemaster-cli --test Connectivity/connectivity03 example.com
+To see all available command line options, use the `--help` command.
 
-For more information on the available tests, you can list them right from
-the command line tool:
+```
+zonemaster-cli --help
+```
 
-    $ zonemaster-cli --list_tests
+## Test reports
 
-**Halt early**
+The severity level of the different messages is CRITICAL, ERROR, WARNING, NOTICE,
+INFO, DEBUG, DEBUG2 or DEBUG3. The default reporting level is NOTICE and higher.
+To change the level of reporting you can use a command line option, e.g
+`--level=INFO` includes level INFO and higher in the report. See
+"[Severity Level Definitions]" for more information on the levels.
 
-If you want to check a domain for some problem, but do not wish to wait
-until all tests are finished you can have Zonemaster exit when the first
-error of a certain severity level has been reached:
+By default the output is formatted as plain text in English (or some other
+language), but other more "technical" output formats are also available with
+options `--raw` and `json`, respectively.
 
-    $ zonemaster-cli --stop_level NOTICE example.com
 
-This will halt execution as soon as a NOTICE or higher message is received.
+## Translation
+
+### In Docker
+
+By default all messages are in English. By using the `--locale=LANG` option
+another language can be selected. Select "LANG" from the table below to have
+Zonemaster translated into that language.
+
+LANG | Language
+-----|---------
+da   | Danish
+en   | English
+fi   | Finnish
+fr   | French
+nb   | Norwegian
+es   | Spanish
+sv   | Swedish
+
+E.g.:
+```sh
+docker run -t --rm zonemaster/cli zonemaster.net --locale=da
+```
+
+An alternative is to set the `LC_ALL` environment variable with correct language
+value when the command is invoked, which can be useful if a shell alias is
+created. E.g.
+```sh
+docker run -e LC_ALL=da -t --rm zonemaster/cli zonemaster.net
```

Comment:
Also passing the LANG / LC_ALL environment without value will take the value from the one defined at run time. 
So `docker run -e LANG -t --rm zonemaster/cli zonemaster.net` will use the default locale defined on the user environment. I don't know if it worth mentioning.

Reply:
I have updated with that information, but maybe too many details?

Reply:
I think it looks fine